test/cli_test.py

```diff
@@ -155,6 +155,44 @@ def test_add_move_remove_album_relative(self):
         self.alt_config['link_type'] = 'relative'
         self.test_add_move_remove_album(absolute=False)
 
+    def test_add_update_move_album(self, absolute=True):
```

Comment:
The option is there because I just copied over the `test_add_move_remove_album` test as a starting point. I guess you're right that there's no real point in also testing relative _and_ absolute links here.